### ISSUE TITLE
Bug: RDLibrary: RipCD crashes when no cd is present

### ISSUE THREAD

**Issue by prometoys** [ElvishArtisan/rivendell#52]
Hi,

when no CD is in the CD-Rom, than RDLibrary crashes, if you try to open RipCD from RDLibrary main window . 

```
terminate called after throwing an instance of 'std::bad_alloc'`
  what():  std::bad_alloc
Aborted
```

This also happens, when a CD was there, you succesfully rip a track and then the CD is ejected automatically.

```
Checking /dev/sr0 for cdrom...
    Testing /dev/sr0 for SCSI/MMC interface
        SG_IO device: /dev/sr0

CDROM model sensed sensed: TSSTcorp DVD-ROM TS-L333A D600 

Checking for SCSI emulation...
    Drive is ATAPI (using SG_IO host adaptor emulation)

Checking for MMC style command set...
    Drive is MMC style
    DMA scatter/gather table entries: 1
    table entry size: 131072 bytes
    maximum theoretical transfer: 55 sectors
    Setting default read size to 27 sectors (63504 bytes).

Verifying CDDA command set...
    Expected command set reads OK.
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
Aborted
```

**Comment by ElvishArtisan** [comment]
Unable to reproduce here.  What are the details of your setup (RD version, distro, architecture)?

**Comment by prometoys** [comment]
Hi, 

we use Rivendell from Tryphon-Repository, Version 2.10.2-2 on Debian Wheezy, amd64. 

Do you need more info. If I can generate a more detailed debug output, let me know. 

Thanks,

Keywan

**Comment by ElvishArtisan** [comment]
Fixed in v2.11.0.
